daft/hudi/pyhudi/table.py

```diff
@@ -163,7 +182,7 @@ def schema(self) -> pa.Schema:
 
     @property
     def is_partitioned(self) -> bool:
-        return self._props.partition_fields == ""
+        return bool(self._props.partition_fields)
```

Comment:
`return self._props.partition_fields == ""` seems a little more explicit rather than relying on bool coercion in Python!

Reply:
there was some changes where `self._props.partition_fields` is now returning list of partition fields